cloudflare/src/endpoints/workers/create_secret.rs

```diff
@@ -0,0 +1,37 @@
+use super::WorkersSecret;
+
+use crate::framework::endpoint::{Endpoint, Method};
+
+/// Create Secret
+/// https://api.cloudflare.com/#worker-create-secret
+pub struct CreateSecret<'a> {
+    /// account id of owner of the script
+    pub account_identifier: &'a str,
+    /// the name of the script to attach the secret to
+    pub script_name: &'a str,
+    /// secert's contents
+    pub params: CreateSecretParams,
+}
+
+impl<'a> Endpoint<WorkersSecret, (), CreateSecretParams> for CreateSecret<'a> {
+    fn method(&self) -> Method {
+        Method::Put
+    }
+    fn path(&self) -> String {
+        format!(
+            "accounts/{}/workers/scripts/{}/secrets",
+            self.account_identifier, self.script_name
+        )
+    }
+    fn body(&self) -> Option<CreateSecretParams> {
+        Some(self.params.clone())
+    }
+}
+
+#[derive(Serialize, Clone, Debug)]
+pub struct CreateSecretParams {
+    /// the variable name of the secret that will be bound to the script
+    pub name: String,
+    /// the string value of the secret
+    pub value: String,
```

Comment:
Are secret values strings or bytes? It feels like the latter would be more proper, no?

Reply:
yah bytes is probably better

Reply:
except you can't precisely send a byte string in a json body...

Reply:
You can serialize them as a string, however within the Rust type system it should still be bytes

Reply:
That would force you to think about how that "string" should be deserialized. There isn't a single way to serialize bytes into a string.

Reply:
This was giving me a hard time. Since it's not a blocker I filed an issue to prioritize later https://github.com/cloudflare/cloudflare-rs/issues/98